extensions/Cake.RSyncCore.yml

```diff
@@ -0,0 +1,17 @@
+Type: Addin
+Name: Cake.RSyncCore
+NuGet: Cake.RSyncCore
+Assemblies:
+- /**/Cake.RSyncCore.dll
+Repository: 
```

Comment:
No repository link... We should hold on publishing this to the website

Reply:
Also this is just a fork of https://cakebuild.net/extensions/cake-rsync/. We should ignore this and encourage to update the original addin instead.

Reply:
@augustoproiete I removed Cake.RSyncCore. Can you please give it another review